### PR TITLE
Create pull_request_template.md

### PR DESCRIPTION
**Pull request by SibingWu**
Create a pull request template for the repository so that there is no need to keep formatting the pull request description

### CODE REVIEW

File: pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Description
+
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+Fixes # (issue)
```
Comment (htthach) [inline pull_request_template.md:5]: Is it "# Fixes" or "Fixes #" ?

File: pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Description
```
Comment (htthach) [inline pull_request_template.md]: Maybe use ## to be the same as the "Type of Change" section below?

File: pull_request_template.md (hunk 1 above)
Comment (SibingWu) [reply pull_request_template.md:5]: I suppose to automatically link the PR to issue is "keyword" + "#issueid"?

File: pull_request_template.md (hunk 2 above)
Comment (SibingWu) [reply pull_request_template.md]: Make sense! I will fix that!

**Review by htthach — APPROVED** [review]

**Pull request merged**